ssl/record/ssl3_record_tls13.c

```diff
@@ -21,7 +21,7 @@
  *   -1: if the record's AEAD-authenticator is invalid or, if sending,
  *       an internal error occurred.
  */
-int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int send)
+int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
```

Comment:
Is it back? It used to issue shadow warning (and that's what it's about, right?), then it stopped. And it's arguably sensible thing to do, i.e. *not* to issue warning for a local variable to shadow function. It makes sense to issue warning is local variable shadows global variable, but not function. And indeed it's only 4.6 that complains, but not 4.8, 5.4 or clang. This actually calls for following question. Warning apparently *was* recognized as bug, you know, since it was fixed, and then do we actually have to play by broken rules?